src/system/user.cr

```diff
@@ -143,7 +127,7 @@ struct System::User
   # :nodoc:
   def self.check_uid_in_bounds(uid : Int) : Nil
     return if (uid >= 0 && uid <= UID_MAX)
-    raise OutOfBoundsError.new("uid: '#{uid}' is out of bounds.")
+    raise OutOfBoundsError.new("uid: '#{uid.inspect}' is out of bounds")
```

Comment:
You should keep quotes out since they are added automatically when calling ‘#inspect’.

Reply:
My bad. forgot to remove them.